<h3 style="margin:0px">Class: org.apache.pig.parser.TestColumnAliasConversion (13 methods) </h3><br><pre style="padding:0"><code class="html"><div id="tags"><button id="10"style="border-width:0;padding:0;background:#282B2E"  onclick="filter('10')" data-toggle="tooltip" title="Executes methods or other tests from the same test unit"><kbd id="tag-10"class="label-info"style="display: inline-block;font-size:7pt" >ExecutionTester&nbsp;(7)</kbd></button>&nbsp;<button id="9"style="border-width:0;padding:0;background:#282B2E"  onclick="filter('9')" data-toggle="tooltip" title="Verifies (un)successful execution of the test case by reporting explicitly a failure"><kbd id="tag-9"class="label-info"style="display: inline-block;font-size:7pt" >UtilityVerifier&nbsp;(6)</kbd></button>&nbsp;</div></code></pre><center><button onclick="showBlocks()" style="margin:6px">Update filter <span class="glyphicon glyphicon-refresh" aria-hidden="true"></span> </button></center><br>
<pre class="type-10 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
@Test public void test7() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x'; " + "B = load 'y' as ( u : int, v : chararray );" + "C = foreach A generate B.$1, $0; "+ "D = store C into 'output';";
  validate(query);
}

</code></pre>

<pre class="type-10 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
@Test public void test6() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x'; " + "B = foreach A generate $1, $1000; " + "C = store B into 'output';";
  validate(query);
}

</code></pre>

<pre class="type-9 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Verifies (un)successful execution of the test case by reporting explicitly a failure">UtilityVerifier</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Verifies (un)successful execution of the test case by reporting explicitly a failure
"></span><br>
@Test public void testInvalidNestedProjection() throws Exception {
  String query="A = load 'x' as (field);" + "B = foreach A {" + "  C = LIMIT invalidName 1;"+ "  generate C.foo;"+ "};";
  try {
    validate(query);
  }
 catch (  PlanValidationException ex) {
    System.out.println(ex.getMessage());
    return;
  }
  Assert.fail("Query should fail to validate.");
}

</code></pre>

<pre class="type-10 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
@Test public void test1() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x' as ( u:int, v:long, w:bytearray); " + "B = foreach A generate u, $1, w; " + "C = store B into 'output';";
  verify(query);
}

</code></pre>

<pre class="type-9 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Verifies (un)successful execution of the test case by reporting explicitly a failure">UtilityVerifier</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Verifies (un)successful execution of the test case by reporting explicitly a failure
"></span><br>
@Test public void testNegative1() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x' as ( u:bag{tuple(x, y)}, v:long, w:bytearray); " + "B = foreach A generate u.(x, $3), v, w; " + "C = store B into 'output';";
  try {
    validate(query);
  }
 catch (  ParserValidationException ex) {
    return;
  }
  Assert.fail("Query should fail to validate.");
}

</code></pre>

<pre class="type-10 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
@Test public void test3() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x' as ( a : bag{ T:tuple(u, v) }, c : int, d : long );" + "B = foreach A { R = a; P = c * 2; Q = P + d; S = R.u; T = limit S 100; generate Q, R, S, T, c + d/5; };" + "store B into 'y';";
  verify(query);
}

</code></pre>

<pre class="type-9 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Verifies (un)successful execution of the test case by reporting explicitly a failure">UtilityVerifier</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Verifies (un)successful execution of the test case by reporting explicitly a failure
"></span><br>
@Test public void testNegative4() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x' as ( u:bag{tuple(x, y)}, v:long, w:bytearray); " + "B = foreach A generate u.z, v, w; " + "C = store B into 'output';";
  try {
    validate(query);
  }
 catch (  ParserValidationException ex) {
    return;
  }
  Assert.fail("Query should fail to validate.");
}

</code></pre>

<pre class="type-10 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
@Test public void test2() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x' as ( u:bag{tuple(x, y)}, v:long, w:bytearray); " + "B = foreach A generate u.(x, y), v, w; " + "C = store B into 'output';";
  verify(query);
}

</code></pre>

<pre class="type-9 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Verifies (un)successful execution of the test case by reporting explicitly a failure">UtilityVerifier</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Verifies (un)successful execution of the test case by reporting explicitly a failure
"></span><br>
@Test public void testNegative3() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x' as ( u:bag{tuple(x, y)}, v:long, w:bytearray); " + "B = foreach A generate u.(x, y), v, x; " + "C = store B into 'output';";
  try {
    validate(query);
  }
 catch (  PlanValidationException ex) {
    return;
  }
  Assert.fail("Query should fail to validate.");
}

</code></pre>

<pre class="type-9 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Verifies (un)successful execution of the test case by reporting explicitly a failure">UtilityVerifier</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Verifies (un)successful execution of the test case by reporting explicitly a failure
"></span><br>
@Test public void testNegative2() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x' as ( u:bag{tuple(x, y)}, v:long, w:bytearray); " + "B = foreach A generate u.(x, y), v, $5; " + "C = store B into 'output';";
  try {
    validate(query);
  }
 catch (  PlanValidationException ex) {
    return;
  }
  Assert.fail("Query should fail to validate.");
}

</code></pre>

<pre class="type-10 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
@Test public void test5() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x' as ( u:bag{} ); " + "B = foreach A generate u.$100; " + "C = store B into 'output';";
  validate(query);
}

</code></pre>

<pre class="type-10 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
@Test public void test4() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x' as ( u:bag{tuple(x, y)}, v:long, w:bytearray); " + "B = foreach A generate u.(x, $1), $1, w; " + "C = store B into 'output';";
  validate(query);
}

</code></pre>

<pre class="type-9 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Verifies (un)successful execution of the test case by reporting explicitly a failure">UtilityVerifier</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Verifies (un)successful execution of the test case by reporting explicitly a failure
"></span><br>
@Test public void testNegative5() throws RecognitionException, ParsingFailureException, IOException {
  String query="A = load 'x';" + "B = foreach A generate u, $1; " + "C = store B into 'output';";
  try {
    validate(query);
  }
 catch (  PlanValidationException ex) {
    return;
  }
  Assert.fail("Query should fail to validate.");
}

</code></pre>

<script>$(document).ready(function() {
  $('pre code').each(function(i, block) {
    hljs.highlightBlock(block);
  });
});</script>
